languages/javascript/exercises/concept/array-analysis/array-analysis.spec.js

```diff
@@ -0,0 +1,93 @@
+import {
```

Comment:
Please update these method names according to the suggestions to the stub file.

- [ ] imports
- [ ] test names
- [ ] test contents